CRM/Upgrade/Incremental/sql/4.7.8.mysql.tpl

```diff
@@ -9,14 +9,3 @@ DELETE FROM civicrm_state_province WHERE name = 'Fernando de Noronha';
 
 -- CRM-17118 extend civicrm_address postal_code to accept full data strings from paypal etc.
 ALTER TABLE civicrm_address CHANGE `postal_code` `postal_code` varchar(64) ;
-
```

Comment:
As long as the remove functions are agnostic it is good to remove from here. ie. DELETE FROM x WHERE y will run whether or not Y exists